pkg/dashboard/ui/src/app/shared/services/import.service.js

```diff
@@ -74,28 +96,109 @@
+                                }
+                            });
                     });
 
-                    if (lodash.isFunction(onSuccess)) {
-                        onSuccess()
-                    }
+                    return $q.all(checkedFunctionList);
                 })
                 .catch(function (error) {
                     DialogsService.alert($i18next.t('common:ERROR_MSG.IMPORT_PROJECT', {lng: lng}))
                         .then(function () {
                             if (lodash.isFunction(onFailure)) {
-                                onFailure(error)
+                                onFailure(error);
                             }
                         });
                 });
         }
+
+        /**
+         * Opens confirm dialog
+         * @returns {Object} closeOptions
+         */
+        function openConfirmDialog(functionName, projectName) {
+            var data = {
+                displayAllOptions: displayAllOptions,
+                dialogTitle: $i18next.t('common:OVERRIDE_FUNCTION_CONFIRM', {lng: lng, functionName: functionName, projectName: projectName})
+            };
+
+            return ngDialog.open({
+                template: '<igz-import-project-dialog ' +
+                    'data-display-all-options="ngDialogData.displayAllOptions" ' +
+                    'data-close-dialog="closeThisDialog({action, option})" ' +
+                    'data-dialog-title="ngDialogData.dialogTitle">' +
+                    '</igz-import-project-dialog>',
+                plain: true,
+                data: data,
+                className: 'ngdialog-theme-iguazio image-dialog'
+            })
+                .closePromise
+                .then(function (closeOptions) {
+                    return closeOptions;
+                });
+        }
+
+        /**
+         * Handles action and option for resolve conflict function
+         */
+        function resolveConflict() {
+            var currentProject = lodash.keys(conflictProjectsData)[0];
+            var currentFunction = lodash.get(conflictProjectsData, [currentProject] + '[0]');
+
+            openConfirmDialog(currentFunction.metadata.name, currentProject)
+                .then(function (data) {
+                    if (data.value.option === 'allProjects') {
+                        lodash.mapKeys(conflictProjectsData, function (functionsList, projectName) {
+                            resolveConflictFunctionsInProject(functionsList, projectName, data.value.action, data.value.option);
+                        });
+                    } else if (data.value.option === 'singleProject') {
+                        resolveConflictFunctionsInProject(conflictProjectsData[currentProject], currentProject, data.value.action, data.value.option);
+                    } else {
+                        resolveConflictFunctionsInProject([currentFunction], currentProject, data.value.action, data.value.option);
+                    }
```

Comment:
Non-critical comment, it's just a matter of coding style, so you can ignore it if you don't relate to this:

Since the only difference between these two statements:
```js
resolveConflictFunctionsInProject(conflictProjectsData[currentProject], currentProject, data.value.action, data.value.option);
```

```js
resolveConflictFunctionsInProject([currentFunction], currentProject, data.value.action, data.value.option);
```

is the first argument you send in the function call, you can first determine what is the value to send, then write the function call just once:

```suggestion
                    } else {
                        var functions = data.value.option === 'singleProject' ? conflictProjectsData[currentProject] : [currentFunction];
                        resolveConflictFunctionsInProject(functions, currentProject, data.value.action, data.value.option);
                    }
```